src/sonic-yang-mgmt/setup.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+from setuptools.command.build_py import build_py
+from os import system
+from sys import exit
+
+# important reuirements parameters
+build_requirements = ['../../target/debs/stretch/libyang_1.0.73_amd64.deb',
+                      '../../target/debs/stretch/libyang-cpp_1.0.73_amd64.deb',
+                      '../../target/debs/stretch/python2-yang_1.0.73_amd64.deb']
+
+install_requirements = []
+
+setup_requirements = ['pytest-runner',]
+
+test_requirements = ['pytest>=3',]
+
+# read me
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+# class for prerequisites to build this package
+class pkgBuild(build_py):
+    """Custom Build PLY"""
+
+    def run (self):
+        # install libyang, it will be used for testing a build time
```

Comment:
for testing at build**t**ime ???

If it is for build time requirement, should it be part of slave docker image?

Reply:
As discussed this can not be part of sonic slave by default because that will install old package in Sonic-slave. We need to install this custom package for our testing, i.e. same package which is built with Sonic-buildimage. Thx.